rental.rb

```diff
@@ -0,0 +1,7 @@
+class Rental
+  attr_accessor :data
+
+  def initialize(_data)
+    @data = date
```

Comment:
- [ ] According to the requirements, you should be initializing ```date``` here and not data, kindly change ```data``` to ```date```.
- [ ] You are currently missing these relationships for rentals
<img width="829" alt="Screenshot 2023-03-18 at 10 34 58" src="https://user-images.githubusercontent.com/64774936/226097655-d9386016-2e6c-402f-95ea-840f79a387f7.png">

You can read more about it [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). It should be something like this:

```
class Rental
  
  attr_accessor :date, :book, :person
 
  
  def initialize(date, book, person)
   
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
end
```

Reply:
resolved! ✔️